includes/templates/template_default/templates/tpl_checkout_confirmation_default.php

```diff
@@ -175,7 +175,7 @@
         ?>
 
         <?php
-        echo zen_draw_form('checkout_confirmation', $formActionLink, 'post', 'id="checkout_confirmation" onsubmit="submitonce();"');
+        echo zen_draw_form('checkout_confirmation', $formActionLink, 'post', 'id="checkout_confirmation" onsubmit="submitonce();"' . (defined('MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE') && MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE == 'InContext' && defined('MODULE_PAYMENT_PAYPALWPP_MERCHANTID')    && MODULE_PAYMENT_PAYPALWPP_MERCHANTID    != '' && defined('MODULE_PAYMENT_PAYPALWPP_STATUS')        && MODULE_PAYMENT_PAYPALWPP_STATUS        == 'True' ? ' data-paypal-button="true" data-paypal-id="' . MODULE_PAYMENT_PAYPALWPP_MERCHANTID . '"':''));

```

Comment:
I'm still struggling over the 2 changes in this file. They're completely PayPal-specific, and it would be preferable to leave payment-module-specific stuff out of the templates.

Thoughts?